components/expander.jsx

```diff
@@ -1,12 +1,15 @@
 var React = require('react');
 
+var AnchorManagerMixin = require('../mixins/anchor-manager');
+
 var Expander = React.createClass({
+  mixins: [AnchorManagerMixin],
   getInitialState: function() {
     return {
       expanded: false
     };
   },
-  collapse: function() {
+  collapse: function(e) {
```

Comment:
Oh, no, I think at some point in coding, I had `collapse` double as an event handler, but then i undid that and forgot to remove the args. Will fix.